src/current/v23.2/ui-network-latency-page.md

```diff
@@ -33,15 +38,32 @@ For instance, the cluster shown above has nodes in `us-west1`, `us-east1`, and `
 
 ### No connections
 
-Nodes that have lost a connection are displayed in a separate color. This can help you locate a network partition in your cluster.
+Nodes that have lost a connection are color-coded depending on connection status: orange with an exclamation mark for `unknown connection state`, and red with a [no symbol](https://www.wikipedia.org/wiki/No_symbol) for `Failed connection`. This can help you locate a network partition in your cluster.
+
+<img src="{{ 'images/v23.2/ui_network_latency_matrix_suspect_node.png' | relative_url }}" alt="DB Console Network Latency suspect node" style="border:1px solid #eee;max-width:100%" />
+
+ Hover over a cell to see:
+
+- the direction of the connection marked by `From` and `To`
```

Comment:
These should also be formatted as sentences.

Reply:
updated